### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by dd483** [kathy908000/Population-Segmentation-For-Health-System-Management#8]
This project is about using New York State hospital data to predict length of stay and total cost for each cancer patient.

The report is well-written, documenting the variables in the dataset in the data preprocessing section, the process for feature selection through the 4 elimination factors, and the preliminary modeling. The team is focused on not overfitting or underfitting, as it is mentioned multiple times throughout the report. Finding the bootstrap variance was a good technique for analyzing overfitting! The team also demonstrates good analysis of the random forest output, specifically the graph of the R^2 iterations.

The report mentions that 4 linear regression models were run, but does not specify the differences in these models. It would be helpful to detail the features included in each, the loss function, and regularizers used, if any, so the reader could distinguish the model differences. Also, cross-validation was said to be used, but the percentage split in training and testing was not specified. Since the report mentions MAE was used for its insensitivity to outliers, the team may want to try different loss functions like Huber that are fit inliers better. Finally, since the random forest section analyzes R^2, maybe R^2 could also be included in the linear regression section for model performance comparison.
